os/drivers/testcase/Make.defs

```diff
@@ -39,6 +39,10 @@ ifeq ($(CONFIG_ARMV8M_TRUSTZONE),y)
 CSRCS += test_tz.c
 endif
 
+ifeq ($(CONFIG_EXAMPLES_STACK_PROTECTION),y)
```

Comment:
Please rebase this file, this is changed to `os/drivers/kernel_test/Make.defs`

Reply:
done